docs/_includes/core-environment-options.csv

```diff
@@ -0,0 +1,28 @@
+Value,Default,Description
+AKKA_HTTP_REQUEST_TIMEOUT,15 minutes,"HTTP request timeout. Should be long, to support large file uploads."
```

Comment:
This is `akka-http` parameter. Better to refer to the akka docs? 

Let's not mention file uploads, as we found problems with setting this high during load test.